src/alto-ui/Form/Switch/Switch.scss

```diff
@@ -64,7 +64,6 @@
 
 .Switch__label {
   line-height: 1;
-  flex: 1;
```

Comment:
you're not finxing anything here, you removing feature. Did you test on Storybook ?